### PR TITLE
Sticky Left Nav

### PR DESCRIPTION
**Pull request by Jeremy1026**
### Relevant Ticket

[LG-9737](https://cm-jira.usa.gov/browse/LG-9737)

[Figma](https://www.figma.com/file/HHD28cToRlcOcqYcrFIm2o/Login.gov-Developer-Guide-(LG-9258%2C-LG-9244)?type=design&node-id=0-1&mode=design)

### Description of Changes

Adds the left nav inline to the top nav when in mobile.

### Screenshots

image

### CODE REVIEW

File: assets/scss/main.css.scss [hunk 1]
```diff
@@ -32,3 +32,9 @@
 .reversefootnote {
   @include u-font-family("mono");
 }
+
+.usa-layout-docs__sidenav {
+  position: sticky;
+  top: 2rem;
+  height: 100vh;
```
Comment (nprimak) [inline assets/scss/main.css.scss:39]: For setting the height we can use a utility class from uswds, `height-viewport`

https://designsystem.digital.gov/utilities/height-and-width/

File: assets/scss/main.css.scss [hunk 2]
```diff
@@ -32,3 +32,9 @@
 .reversefootnote {
   @include u-font-family("mono");
 }
+
+.usa-layout-docs__sidenav {
+  position: sticky;
```
Comment (nprimak) [inline assets/scss/main.css.scss:37]: For setting the position we can use a uswds utility class, `position-sticky`

https://designsystem.digital.gov/utilities/display/#position

**Review by nprimak — CHANGES_REQUESTED** [review]
Since we're using USWDS for the dev docs and trying to keep custom css as minimal as possible, lets use the utility classes whenever possible

File: assets/scss/main.css.scss (hunk 2 above)
Comment (Jeremy1026) [reply assets/scss/main.css.scss]: Updated to use the USWDS class

File: assets/scss/main.css.scss (hunk 1 above)
Comment (Jeremy1026) [reply assets/scss/main.css.scss]: Updated to use the USWDS class

**Review by nprimak — APPROVED** [review]
Looking good!

**Review by anniehirshman-gsa — COMMENTED** [review]
Mobile menu visually LGTM, thank you!

One comment: the sidebar is shown at the bottom of the page as well, which creates a big gap between the end of the main content and the footer/return to top/edit this page buttons. Could we hide the aside at mobile breakpoints at the bottom of the page, but keep it as-is in the mobile menu?

Screen Shot 2023-10-12 at 1 24 40 PM

**Review by anniehirshman-gsa — APPROVED** [review]
LGTM thank you!!

**Pull request merged**